### DISCUSSION TITLE
Tracking only one channel of a multiple-channel image

### DISCUSSION THREAD

**Discussion by irelamb** [BBQuercus/deepBlink#148] [Q&A] [answered]
Can we give as input a 2-channel image and ask deepBlink to track spots only in one channel?

Thanks!

Irene

**Comment by BBQuercus** [comment, marked as answer]
So far we only have the functionality to specify which axis are channels and then detect all of them. If you have a higher throughput experiment it's usually easier to run it through the pythonic API directly where you can specify the channels. Let me know if you need help with writing a short script to allow for this.

**Reply by irelamb** [reply]
Hi Bastian, thank you again for your answer; it's indeed what I did. A detail: your `load_image` function also reads .ome.tif files, which is great for me, maybe you can add it in its description :)
I was using the ms2.h5 model to track mRNAs tagged with an MS2 system in far red (confocal spinning-disk microscope, EMCCD camera, 60x oil objective). I works fairly well but not as good as TrackMate on my data. I thought I could train a model myself since I have a lot of data that are already labelled with TrackMate. My data are in the form of time lapses, and I was wondering what was best to do to train the model... Should I use multiple frames (far apart one to the other, for example the beginning, the middle, and the end of the video) from different experiments so to maximise the variability contained in the data? For example, in the end of the video is usually harder to detect spots because of photobleaching...

Maybe one last comment, despite the package is very well structured and easy to use I think it would profit a lot of a Jupyter notebook that shows how to use it from a Python interface!

**Reply by BBQuercus** [reply]
Hey,

Thanks for the feedback 👍. I think a notebook beyond the more advanced examples might be useful.

Regarding training, I'd say a reasonably diverse set works pretty well - we've also trained some models with data from the same time/z stacks and usually take some random indexes that are far enough apart that the (almost) same images don't appear in both the training and testing set (i.e. just far enough so the spots are no longer in the same position).

Otherwise if the MS2 works reasonably, you can use it as pre-trained model and put this in the configuration file during training. Also to note, our MS2 model was only labeled on cytoplasmic spots to not get a nucleus full of spots (via NLS) - depending on your pre/post processing.
